src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,28 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.info("Method-function completeOrder() was called. Params: userId = {}", userId);
```

Comment:
There is a better logger method to use when method invocation

Reply:
debug?

Reply:
fix it all over the place pls

Reply:
getAllProductsFromShoppingCart() too?

Reply:
or i can leave it as info?

Reply:
except for this one c: We need to know if all the  data was successfully fetched